cardea/problem_definition/prolonged_length_of_stay.py

```diff
@@ -0,0 +1,150 @@
+import featuretools as ft
+import pandas as pd
+
+from cardea.problem_definition import ProblemDefinition
+
+
+class LengthOfStay (ProblemDefinition):
+    """Defines the problem of length of stay, finding how many days
+        the patient will be in the hospital.
+
+        Attributes:
+        target_label: The target label of the prediction problem.
+        target_entity: The entity name in which it contains the target label.
+        cutoff_time_label: The cutoff time label of the prediction problem
+        cutoff_entity: The entity name in which it contains the cutoff time label.
+        prediction_type: The type of the machine learning prediction.
+        """
+
+    updated_es = None
+    target_label = 'length'
+    target_entity = 'Encounter'
+    cutoff_time_label = 'start'
+    cutoff_entity = 'Period'
+    prediction_type = 'classification'
+
+    def __init__(self, t=7):
+        self.threshold = t
+
+    def generate_cutoff_times(self, es):
+        """Generates cutoff times for the predection problem.
+
+            Args:
+            es: fhir entityset.
+
+            Returns:
+            es, target_entity, Series of target_label and a dataframe of cutoff_times
+
+            Raises:
+            ValueError: An error occurs if the cutoff variable does not exist.
+            """
+        try:
+            if (self.check_target_label(es, self.target_entity, self.target_label)):
+                if not (self.check_target_label_values(es, self.target_entity, self.target_label)):
```

Comment:
@ShahadThobaiti I notice there is no else associated with these two if statements.  It should be fine because `check_target_label` and `check_target_label_value` raise an error if they do not return 'true' and 'false' respectively.  I just want to make sure you've thought that through and did it that way intentionally.

Reply:
the methods were initially returning true/false. However, I changed it to raise an error when it is false. Do you recommend removing the if statements as there is no else associate with?